prime-router/src/main/kotlin/azure/observability/event/ReportStreamEventService.kt

```diff
@@ -275,6 +295,28 @@ class ReportStreamEventService(
         ).send()
     }
 
+    override fun sendReceiveProcessingError(
+        eventName: ReportStreamEventName,
+        pipelineStepName: TaskAction,
+        error: String,
+        reportId: ReportId,
```

Comment:
I can't use `sendReportProcessingError` since it uses `childReport`. This event is used when sender is not found. In that case a report cannot be created. I can change `reportId` to `submissionId` to better delineate though.